PhysicsTools/PatAlgos/python/tools/pfTools.py

```diff
@@ -316,10 +316,10 @@ def switchToPFJets(process, input=cms.InputTag('pfNoTauClones'), algo='AK4', pos
     print "input collection: ", input
 
     if algo == 'AK4':
-        genJetCollection = cms.InputTag('ak4GenJetsNoNu'+postfix)
+        genJetCollection = cms.InputTag('ak4GenJets'+postfix)
```

Comment:
Yes, I noticed that, this was a mistake on my part. I will revert it to the way it was in the past.